OpenRA.Game/Game.cs

```diff
@@ -905,5 +912,23 @@ public static bool SetClipboardText(string text)
 		{
 			return Renderer.Window.SetClipboardText(text);
 		}
+
+		public static void BenchmarkMode(string prefix)
+		{
+			benchmark = new Benchmark(prefix);
+		}
+
+		public static void LoadMap(string launchMap)
+		{
+			var orders = new List<Order>
+			{
+				Order.Command("option gamespeed {0}".F("default")),
+				Order.Command("state {0}".F(Session.ClientState.Ready))
+			};
+
+			var map = ModData.MapCache.SingleOrDefault(m => m.Title == launchMap);
```

Comment:
We generally specify maps by their UID or path.
Can we please change this to
```
var path = Platform.ResolvePath(launchMap);
var map = ModData.MapCache.SingleOrDefault(m => m.Uid == launchMap) ??
	ModData.MapCache.SingleOrDefault(m => m.Package.Name == path);
``` 
?